README.md

```diff
@@ -1,5 +1,11 @@
 # Postgres Test
 
-This repo contains one branch per language for the cockroach acceptance tests.
-These are auto-built on push and tagged on the docker hub at
-https://hub.docker.com/r/cockroachdb/postgres-test/.
+This is the Docker image used to run the CockroachDB
+[acceptance tests](https://github.com/cockroachdb/cockroach/tree/master/acceptance),
+which include verification that simple clients in various languages can connect
+to and interact with CockroachDB using the Postgres wire protocol -- thus it
+includes the tools required to run php, node, java, etc.
+
+[Docker Hub](https://hub.docker.com/r/cockroachdb/postgres-test/) automatically
+builds new images from git tags, using the git tag as the Docker tag. By
+convention, tags are of the form YYYYMMDD-HHMM.
```

Comment:
I thought about it, but opted to just link to the acceptance package (above) rather than something too specific, that will inevitably rot, since the wrong answer is easily worse than no answer.